frontend/javascripts/oxalis/controller/viewmodes/plane_controller.js

```diff
@@ -172,31 +172,27 @@ function createDelayAwareMoveHandler(multiplier: number) {
     const state = Store.getState();
     let direction = Math.sign(multiplier);
 
-    const moveDistanceIn1Second =
-      state.userConfiguration.moveValue / getBaseVoxel(state.dataset.dataSource.scale);
     const { activeViewport } = state.viewModeData.plane;
+    const thirdDim = dimensions.thirdDimensionForPlane(activeViewport);
+    const voxelPerSecond =
+      state.userConfiguration.moveValue / state.dataset.dataSource.scale[thirdDim];
```

Comment:
@daniel-wer Using `getBaseVoxelFactors` didn't produce the correct result (the timing was still off). It turned out that it was even easier: just use the nm-based moveValue and divide it by the correct component of the dataset scale. Now, the delay should be right :)